packet_testing/extclass_packet_testing.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import ipaddress
+import scapy.all
+from scapy.layers.inet import _IPOption_HDR
+from scapy.layers import inet6
+
+
+parser = argparse.ArgumentParser(description='Tests the packets used during the MCA extended classification step.')
```

Comment:
Wrap these and the code at the bottom inside `if __name__ == "__main__"`, otherwise it will execute on `import`.